style/modules/textRight.css

```diff
@@ -0,0 +1,5 @@
+.textRight{
+    text-align: right;
+    line-height: 35px;
+    
```

Comment:
there's an empty line to remove here, and also a missing newline (but I think when you save it again that will get fixed automatically).